api/types/types_model.go

```diff
@@ -139,28 +169,28 @@ type Volume struct {
 	IOPS int64 `json:"iops,omitempty" yaml:"iops,omitempty"`
 
 	// The name of the volume.
-	Name string `json:"name"`
+	Name string `json:"name" yaml:"name,omitempty"`
```

Comment:
Oh, because I got Copy/Paste happy with the YAML update. Doesn't matter ultimately because the YAML tags were originally there to influence the CLI output which is no longer based on YAML.